lib/Settings/User.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * @copyright Copyright (c) 2017 Robin Appelman <robin@icewind.nl>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\GroupFolders\Settings;
+
+use OCP\IUserSession;
+use OCP\IGroupManager;
+use OCP\Settings\ISettings;
+use OCP\AppFramework\Http\TemplateResponse;
+use OCA\GroupFolders\Service\DelegationService;
+
+class User implements ISettings {
```

Comment:
Same here

If I remember the general workspace manger was a normal delegated admin, and the admin with lower access ( don't remember the name) won't see the settings page anyway, so we shouldn't need more settings page

Reply:
In our workspace app, we have 2 groups to manage the workspaces :

1. GeneralManager : Users can use all API/REST from the Groupfolders apps and use its interface

2. WorkspacesManagers : Users can use all API/REST only. But, they are limited in our app. They don't define the quota, create a groupfolder, list all groupfolders (they can list groupfolders with they can manage only). We don't know if in the future they can define the quota, create a groupfolder or list all groupfolders.

So, yes, I think I don't need more settings page :+1:

Reply:
Done ! :ok_hand: